src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/EventLogEntry.cs

```diff
@@ -420,7 +420,7 @@ private string GetMessageLibraryNames(string libRegKey)
                         result.Append(owner.MachineName);
                         result.Append(@"\");
```

Comment:
Looks like one got missed?

Reply:
Yup.  Incomplete regex :)  Hopefully when actually implemented in https://github.com/dotnet/runtime/issues/30740, that rule will do better. (One of my motivations here was to see how common this actually was, to help understand the benefit of such a rule.)